Sources/StripeKit/Core Resources/PaymentIntents/PaymentIntent.swift

```diff
@@ -76,11 +76,16 @@ public struct StripePaymentIntent: StripeModel {
     /// Status of this PaymentIntent, one of `requires_payment_method`, `requires_confirmation`, `requires_action`, `processing`, `requires_capture`, `canceled`, or `succeeded`.
     public var status: StripePaymentIntentStatus?
     /// The data with which to automatically create a Transfer when the payment is finalized. See the PaymentIntents Connect usage guide for details.
-    public var transferData: [String: String]?
+    public var transferData: TransferData?
     /// A string that identifies the resulting payment as part of a group. See the PaymentIntents Connect usage guide for details.
     public var transferGroup: String?
 }
 
+public struct TransferData: StripeModel {
```

Comment:
Can we prefix this types name with `StripePaymentIntent` so it’ll be `StripePaymentIntentTransferData`. This way it’ll be clear when using this type what it’s associated with. 

Also let’s add some [documentation](https://stripe.com/docs/api/payment_intents/object#payment_intent_object-transfer_data) above each property so their usage can be clear.

Reply:
It makes sense.